homeassistant/components/yeelight/__init__.py

```diff
@@ -65,7 +67,7 @@
 
 NIGHTLIGHT_SWITCH_TYPE_LIGHT = "light"
 
-SCAN_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=600)
```

Comment:
We should be able to drop the polling completely since there is auto reconnect

Reply:
Schouldn't we keep the polling at a low rate (1x per 10 minutes) to ensure the state will never go out of sync if the push messages are sometimes not coming through?
I like to keep it as an extra ensurence the state stays in sync.

Reply:
The push is tcp and has auto reconnect, I don't think we are worried about lost packets since it's not udp, unless there is a known bug in the Yeelight code that stops sending state, we shouldn't poll as well.

Reply:
I just removed the polling